main.py

```diff
@@ -1,7 +1,36 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+import init_django_orm  # noqa: F401
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ("Western", "Action", "Dramma")
+    actors = [
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Scarlett", "Keegan"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"],
+        ["Scarlett", "Johansson"],
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        Actor.objects.create(first_name=actor[0], last_name=actor[1])
```

Comment:
either use tuples in `actors` or use `for name, surname in actors:` in order to avoid indexes

Reply:
Done